pom.xml

```diff
@@ -451,6 +451,11 @@
         <artifactId>stagemonitor-web</artifactId>
         <version>${stagemonitor.version}</version>
       </dependency>
+      <dependency>
+        <groupId>org.tukaani</groupId>
+        <artifactId>xz</artifactId>
```

Comment:
Interesting name. I think we should start naming our artifacts like this. ‘Just add a dependency on ab3 and you’re good to go’

Reply:
I think you're on to something here. Apparently `xz` is a compression algorithm so maybe the letters actually mean something.